src/components/discover-sheet/RegisterENSSection.tsx

```diff
@@ -27,6 +31,13 @@ export default function RegisterENSSection() {
   const { navigate } = useNavigation();
   const { colors } = useTheme();
   const { isReadOnlyWallet } = useWallets();
+  const { uniqueDomain } = useAccountENSDomainGroups();
+
+  useEffect(() => {
+    if (uniqueDomain?.name) {
+      prefetchENSProfileRecords({ name: uniqueDomain.name });
```

Comment:
wat dis?

Reply:
in the ens intro screen, the text "Edit notben.eth" vs "Use notben.eth" depends on if notben.eth has any records set yet. I'm prefetching the records on the discover sheet so there's no lag when determining the text usage when first rendering the ens intro screen